programs/lz4io.h

```diff
@@ -54,6 +54,14 @@ static const char nulmark[] = "/dev/null";
 
 typedef struct LZ4IO_prefs_s LZ4IO_prefs_t;
 
+typedef struct {
+  LZ4F_frameInfo_t frameInfo;
+  char* fileName;
```

Comment:
unless you intend to manage `fileName` buffer lifetime from this pointer, it's likely better to make this pointer `const char*`.